src/lambda/vue/samInvokeVue.ts

```diff
@@ -109,61 +184,92 @@ export const Component = Vue.extend({
             },
             deep: true,
         },
-        payload: function (newval: string) {
-            this.resetJsonError()
+        payload: function (newval: {value: string, errorMsg: string}) {
```

Comment:
Just so I'm clear, how is this errorMsg set?

Reply:
errorMsg is set when attempting to parse the string into JSON.  This happens during save or invoke.